src/auth-service/config/firebase-admin.js

```diff
@@ -2,8 +2,50 @@ const admin = require("firebase-admin");
 const constants = require("./constants");
 const { initializeApp } = require("firebase-admin/app");
 const serviceAccount = require(`${constants.GOOGLE_APPLICATION_CREDENTIALS}`);
+const functions = require("firebase-functions");
+const joinUtil = require("../utils/join");
+const { logObject } = require("../utils/log");
 
 initializeApp({
   credential: admin.credential.cert(serviceAccount),
   databaseURL: constants.FIREBASE_DATABASE_URL,
 });
+
+exports.newUserSignUp = functions.auth.user().onCreate(async (user) => {
+  try {
+    const email = user.email;
+    const message = `Welcome to AirQo, ${user.firstName}`;
+    const subject = "Welcome to AirQo!";
+
+    admin.firestore().collection("users").doc(user.uid).set({
+      email: user.email,
+    });
```

Comment:
Hi @Baalmart this shall handled by the frontend for now. You can delete this to avoid our database from growing in size with unused data.

Reply:
Alright @NoahNsimbe , this is noted.